SpanishFilter/sections/adservers.txt

```diff
@@ -4,6 +4,7 @@
 ! https://forum.adguard.com/index.php?threads/ivoox-com-podcasts-issues.29530/
 ! ||addevweb.com^$third-party
 !
+||engagedhits.com^$third-party
```

Comment:
not used on Spanish sites only
https://publicwww.com/websites/%22engagedhits.com%22/

Reply:
Ok I will add it to base.